internal/install/command.go

```diff
@@ -55,7 +54,12 @@ var Command = &cobra.Command{
 		c, err := client.NewClient(configAPI.GetActiveProfileName())
 		if err != nil {
 			// An error was encountered initializing the client.  This may not be a
-			// problem since many commands don't require the use of an initialized client
+			// problem since many commands don't require the use of an initialized client			sg := initSegment()
+			sg := initSegment()
+			defer func() {
+			  sg.Close()
+				time.Sleep(5 * time.Second)
```

Comment:
@NRhzhao this seems excessive, I don't think a user is going to wait 5s if they already cancelled the execution. Isn't there a way to trigger a synchronous flush? Wouldn't the close do that? Otherwise I would not recommend waiting more than 1s at most.

Reply:
@Julien4218 I run the test, and it does wait even in context cancel. The segment runs on a 5-second timer to flush, and I can reduce the timer, but run the risk of losing the message.